src/web/command_control/client/components/CommandControl.tsx

```diff
@@ -1078,6 +1078,34 @@ export default class CommandControl extends React.Component {
 		this.missionHistory.push(deepcopy(mission))
 	}
 
+	/**
+	 * Provides current mission to test if task type has changed between the previous and latest missions
+	 * @returns {void}
+	 * 
+	 * @notes
+	 * Does not track changes in task parameters, only in task type. 
+	 */
+	taskChangeTester() {
```

Comment:
Can we update this function name to be detectTaskChange instead of taskChangeTester.